data-lineage/src/main/java/com/wipro/ats/bdre/lineage/dataaccess/LineageDao.java

```diff
@@ -45,7 +45,7 @@ protected static Connection getHiveJDBCConnection(String dbName) throws SQLExcep
 				Class.forName(IMConstant.HIVE_DRIVER_NAME);
 //				String hiveConnectionString = MDConfig.getProperty("hive.hive-connection", null);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)